### PR TITLE
Add Compressor.io to organisations using fastify

### PR DESCRIPTION
**Pull request by winstef**
Added [Compressor.io](https://compressor.io) logo.
Currently using Fastify V3, thanks!

### CODE REVIEW

**Review by delvedor — APPROVED** [review]
LGTM

File: src/website/data/organisations.yml [hunk 1]
```diff
@@ -54,6 +54,10 @@
   image: "codecks.svg"
   link: "https://www.codecks.io/"
 
+- name: "Compressor"
+  image: "compressor.svg"
+  link: "https://www.compressor.io/"
```
Comment (StarpTech) [inline src/website/data/organisations.yml]: `https://www.compressor.io/` doesn't work because it points to `192.0.2.1` :smile:

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: src/website/data/organisations.yml (hunk 1 above)
Comment (StarpTech) [reply src/website/data/organisations.yml]: Use `https://compressor.io/`

Comment (winstef) [reply src/website/data/organisations.yml]: oops, my bad, fixed in next commit.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**